ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal-network.tsx

```diff
@@ -221,63 +235,150 @@ export const AssetPickerModalNetwork = ({
         <Box
           className="multichain-asset-picker__network-list"
           display={Display.Flex}
+          flexDirection={FlexDirection.Column}
         >
+          {testNetworks.length > 0 ? (
+            <Text
+              variant={TextVariant.bodyMd}
+              color={TextColor.textAlternative}
+              padding={4}
+            >
+              {t('enabledNetworks')}
+            </Text>
+          ) : null}
           <Box
             display={Display.Flex}
             flexDirection={FlexDirection.Column}
             width={BlockSize.Full}
           >
-            {networksList.map((networkConfig) => {
-              const { name, chainId } = networkConfig;
-              return (
-                <NetworkListItem
-                  key={chainId}
-                  name={
-                    NETWORK_TO_SHORT_NETWORK_NAME_MAP[
-                      chainId as keyof typeof NETWORK_TO_SHORT_NETWORK_NAME_MAP
-                    ] ?? name
-                  }
-                  selected={
-                    // If multiselect is enabled, the checkbox indicates selection
-                    isMultiselectEnabled ? false : network?.chainId === chainId
-                  }
-                  onClick={() => {
-                    if (isMultiselectEnabled) {
-                      handleToggleNetwork(chainId);
-                      return;
+            {(testNetworks.length > 0 ? nonTestNetworks : networksList).map(
```

Comment:
nonTestNetworks will always be displayed here so this can be simplified

```suggestion
            {nonTestNetworks.map(
```